lib/game/board.ex

```diff
@@ -56,15 +56,14 @@ defmodule Game.Board do
     trunc(:math.sqrt(Enum.count(board)))
   end
 
-  @spec calculate_turn(map()) :: String.t()
+  @spec calculate_turn(map()) :: atom()
```

Comment:
```suggestion
  @spec calculate_turn(map()) :: :player_one | :player_two
```

Since this only returns either `:player_one` or `:player_two` I think you could tighten up this typespec even more.